opaque_keys/edx/tests/test_aside_keys.py

```diff
@@ -2,33 +2,111 @@
 Tests of AsideUsageKeyV1 and AsideDefinitionKeyV1.
 """
 
-import itertools
 from unittest import TestCase
 
 from six import text_type
 import ddt
+from hypothesis import strategies, given, assume
 
-from opaque_keys.edx.asides import AsideUsageKeyV1, AsideDefinitionKeyV1, _encode, _decode
+from opaque_keys.edx.asides import (
+    AsideUsageKeyV1, AsideDefinitionKeyV1, _encode_v1, _decode_v1,
```

Comment:
There are tests that use AsideUsageKeyV1 and AsideDefinitionKeyV1, but no equivalent specific tests using AsideUsageKeyV2, AsideDefinitionKeyV2. Would it be worthwhile to update or duplicate them?
See for example: test_usage_round_trip_deserialized